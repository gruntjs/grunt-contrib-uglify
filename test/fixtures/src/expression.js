function(doc) {
    emit(doc._id,doc);
}
