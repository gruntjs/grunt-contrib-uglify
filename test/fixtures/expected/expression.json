{employees:[{firstName:"John",lastName:"Doe"}]}